src/main/java/mServer/crawler/sender/arte/ArteHttpClient.java

```diff
@@ -27,14 +28,31 @@
             OkHttpClient httpClient = mvhttpClient.getHttpClient();
             Request request = new Request.Builder()
                     .addHeader(AUTH_HEADER, AUTH_TOKEN)
+                    .addHeader("User-Agent", "Fiddler")
                     .url(aUrl).build();
-            try (Response response = httpClient.newCall(request).execute()) {
-                if(response.isSuccessful()) {
-                    result = gson.fromJson(response.body().string(), aDtoType);
-                } else {
-                    logger.error(String.format("ARTE Request '%s' failed: %s", aUrl, response.code()));
-                }
-            }
+            
+            boolean stop = false;
+            
+            do {
+              Call call = httpClient.newCall(request);
+              try (Response response = call.execute()) {
+                  if(response.isSuccessful()) {
+                      result = gson.fromJson(response.body().string(), aDtoType);
+                      stop = true;
+                  } else {
+                      if(response.code() != 429) {
+                        logger.error(String.format("ARTE Request '%s' failed: %s", aUrl, response.code()));
+                        stop = true;
+                      } else {
+                        // bei 429 (too many requests) warten und nochmal versuchen
+                        try {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Extract this nested try block into a separate method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1141)